app/ci_program.py

```diff
@@ -121,7 +121,7 @@ def commandLoop(self):
               # The sequence was empty, so it looks like this Escape wasn't
               # really the start of a sequence and is instead a stand-alone
               # Escape. Just forward the esc.
-              ch = curses.ascii.ESC
+              ch = app.curses_util.KEY_ESCAPE
```

Comment:
I've switched it back to curses.ascii.ESC for now. It maybe slower to access the library, instead of our stored one, but maybe that's just early optimization.